apicast/config/cloud_hosted.lua

```diff
@@ -2,12 +2,14 @@ local PolicyChain = require('apicast.policy_chain')
 local policy_chain = context.policy_chain
 
 if not arg then -- {arg} is defined only when executing the CLI
+  policy_chain:insert(PolicyChain.load_policy('cloud_hosted.metrics', '0.1', { log_level = 'warn' }))
```

Comment:
For now it records warnings too but for production we probably want it to be `error` and above. Maybe an env var?

Reply:
yes, log_level should be manage by a ENV var.

This would be a policy specific ENV var?

ex: PROMETHEUS_LOG_LEVEL or system wide?